Makefile

```diff
@@ -48,24 +48,16 @@ test-coregeth: \
 hera:
 	./build/hera.sh
 
-ssvm:
-	./build/ssvm.sh
-
 evmone:
 	./build/evmone.sh
 
-aleth-interpreter:
-	./build/aleth-interpreter.sh
-
 # Test EVMC support against various external interpreters.
-test-evmc: hera ssvm evmone aleth-interpreter
+test-evmc: hera evmone
```

Comment:
:notebook: EVMC support/development tooling for `aleth` and `ssvm` is dropped because they don't have an EVMCv10 version yet.

Reply:
So you say that we can bring them soon with #507?